source/Constants.xaml

```diff
@@ -5,6 +5,9 @@
     <!--region ThemeModifier variables-->
     <sys:Double x:Key="DetailsViewDescriptionWidth">300</sys:Double>
     <sys:Boolean x:Key="DetailsViewDescriptionDockToRight">false</sys:Boolean>
+    <VerticalAlignment x:Key="DetailsViewBackgroundVerticalAlignment">Center</VerticalAlignment>
```

Comment:
Regarding the new keys, I think it would make sense to add variables for grid view too